registry/static/html/mods_find.html

```diff
@@ -33,15 +33,17 @@ <h3>find modules</h3>
                 no results
             </span>
         {{else}}
+        <form method="GET" id="populate-new" action="/mods/new">
         <table> <!-- outer list of things -->
             {{range .Found}}
+                {{$txt := .Text}}
```

Comment:
Doesn't seem like we need this extra indirection?

Reply:
it's needed for inside the for loop:
```go
{{range $i, $tag := .Result.Tags}}
```

but maybe I just don't know go templates well enough, on how to reference a variable from the external context.